.rubocop_todo.yml

```diff
@@ -4,3 +4,7 @@ Style/DoubleNegation:
   Enabled: false
 Style/LineLength:
   Max: 110
+Style/MethodLength:
+  Max: 15
```

Comment:
Woot! :)

This is the thing with failing the build through rubocop, we need to relax the rules. A much better rule is `5`, with some agreed upon exceptions, that are exceptional. I wouldn't commit this, and either stop failing the build, or ignore the build for this PR when it's only that rule.